assets/javascripts/src/components/pages/Contribution.jsx

```diff
@@ -123,7 +125,7 @@ class FormWithRecurring extends React.Component {
 
 class FormWithoutRecurring extends React.Component {
     render() {
-        return <div className={'contribute-controls contribute-fields ' + (this.props.amounts.length % 3 ? 'option-button__fours' : 'option-button__three')}>
+        return <div className="contribute-controls contribute-fields option-button__fours">
```

Comment:
It doesn't look like `option-button__fours` is defined anywhere in our css so it should be safe to get rid of it here